src/main/java/com/urswolfer/gerrit/client/rest/http/changes/ListChangesOptionByVersion.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2020 GerritForge Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.urswolfer.gerrit.client.rest.http.changes;
+
+import com.google.gerrit.extensions.client.ListChangesOption;
+
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+class ListChangesOptionByVersion {
+  static final Map<String, ListChangesOption> maxChangeOptionByVersion = new HashMap<String, ListChangesOption>();
+
+  static {
+    put("2.6", ListChangesOption.DETAILED_ACCOUNTS);
```

Comment:
Just noticed that there is exactly one new option per Gerrit release? This is interesting :) If that would not be the case, I think it would make sense to swap key and value of the map.

Have you checked the source when which operation got introduced? Or is there another source?

Reply:
> Just noticed that there is exactly one new option per Gerrit release? This is interesting :) If that would not be the case, I think it would make sense to swap key and value of the map.

Not necessarily: two versions could share the same limit.

> Have you checked the source when which operation got introduced? Or is there another source?

Yep.

Reply:
> > Just noticed that there is exactly one new option per Gerrit release? This is interesting :) If that would not be the case, I think it would make sense to swap key and value of the map.
> 
> Not necessarily: two versions could share the same limit.

Ah, I get it now: new options get added at the end of the enum (with a higher value).

> > Have you checked the source when which operation got introduced? Or is there another source?
> 
> Yep.